features/support/env.rb

```diff
@@ -40,6 +40,11 @@
 
 World(FactoryBot::Syntax::Methods)
 
+Before do
+  User.current_user ||= FactoryBot.create(:user, :supervisor)
```

Comment:
What if we want to test features specific to some other role where we want to write some kind of negative test cases about what specific role cant do.

A negative test case is just an example. I think we need to override it in test cases where we want to use it.

Just adding this comment so I will understand it more.

Reply:
Still haven't figured out how to do per-feature Before statements